docs/en/glossary/glossary.asciidoc

```diff
@@ -551,6 +551,20 @@ nodes handling the user requests.
 endif::cloud-terms[]
 ifdef::elasticsearch-terms[]
 
+[[glossary-recovery]] recovery ::
+The process of loading an index shard from disk, making the shard available for queries.
```

Comment:
We've been saying "shard copy" more than "index shard", mostly because it implies that each shard copy is pretty much the same.

I don't think "from disk" is super accurate. I might say something like "The process of bringing a shard copy up to date and making it available for queries." When we do it on startup we do it by recovering what we have locally and then pulling changes from other shard copies on other nodes. When we do it for replication we start the recovery with a "copy all the files" step. I guess *where* we are doing the recovery from is really interesting which is why I'm stuck on the "from disk" part.

Reply:
Thanks @nik9000. I think I was too narrowly focused on startup there.

I'll submit another commit to update that sentence to `The process of updating a shard copy and making it available for queries.`

It's also makes it clearer that making the shard available is a step of recovery, not just an outcome.